### DISCUSSION TITLE
On MacOS, launching Quasar-Framework/Capacitor dev command targeting Android or iOs fail to open IDE

### DISCUSSION THREAD

**Discussion by jimmyl-cgair** [quasarframework/quasar#14049] [CLI - Capacitor / Cordova modes] [answered]
My project is working well on **Windows 10** environement. But, on **macOS**, only **SPA** and **PWA** are working. **Capacitor Android** and **Capacitor iOS** arent not working. 

Using Quasar following commands fallback to prompt without any further actions. IDE is not launching as it should.

`    quasar dev -m capacitor -T android`
or

`    quasar dev -m capacitor -T ios`
    
Nothing, no IDE starting, not even an error message. Just getting back to the command prompt. What is wrong? What shoud I do?

**build** commands for android or iso are working.

```
 Dev mode.................. capacitor
 Pkg quasar................ v2.7.5
 Pkg @quasar/app-webpack... v3.5.7
 Pkg webpack............... v5
 Debugging................. enabled
 Running mode...... IDE (manual)

Operating System - Darwin(20.6.0) - darwin/x64
NodeJs - 18.6.0

Global packages
  NPM - 8.13.2
  yarn - Not installed
  @quasar/cli - 1.3.2
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.7.5 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-webpack - 3.5.7 -- Quasar Framework App CLI with Webpack
  @quasar/extras - 1.14.3 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.37 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.1.2
  pinia - Not installed
  vuex - Not installed
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  @babel/core - 7.18.9 -- Babel compiler core.
  webpack - 5.73.0 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 4.9.2 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - 6.5.3 -- A plugin for your Webpack build process, helping you generate a manifest of local files that workbox-sw should precache.
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  typescript - 4.5.5 -- TypeScript is a language for application scale JavaScript development
  @capacitor/core - 3.6.0 -- Capacitor: Cross-platform apps with JavaScript and the web
  @capacitor/cli - 3.6.0 -- Capacitor: Cross-platform apps with JavaScript and the web
  @capacitor/android - 3.6.0 -- Capacitor: Cross-platform apps with JavaScript and the web
  @capacitor/ios - 3.6.0 -- Capacitor: Cross-platform apps with JavaScript and the web

Quasar App Extensions
  *None installed*

Networking
  Host - MacBook-Pro-de-RD.local
  en0 - 192.168.1.86
```

**Comment by selfee-jp** [comment]
hello.
Right now, I'm also making an app with Capacitor.
Do you have android studio and xcode installed?

If installed, have you tried the following command?

`quasar dev -m capacitor -T android --ide`
or
`quasar dev -m capacitor -T ios --ide`

It works fine in my environment.

**Reply by jimmyl-cgair** [reply]
of course I have Android Studio and XCode installed. I did found what was wring, see the [following reply](https://github.com/quasarframework/quasar/discussions/14049#discussioncomment-3260173). Thank you to had try something with me.

**Comment by jimmyl-cgair** [comment, marked as answer]
I found the problem and apply a solution. I was copying the project from a Windows10 machine into a MacBookPro. So, permission and execution attribute wasn't on these file. I got to do a chmod. I did that command and the project starts working like it should.

`chmod +x /src-capacitor/node_module@capacitor/cli/bin/capacitor`
